static/src/javascripts/projects/common/modules/identity/upsell/account-creation/AccountCreationFlow.js

```diff
@@ -0,0 +1,208 @@
+// @flow
+/* eslint-disable react/no-danger */
```

Comment:
💥 🤘 

Could we only enable danger on a single line? If danger is unavoidable...